README.md

```diff
@@ -58,6 +58,15 @@ with VNC:
 
 ```$ make image-vnc```
 
+It is also possible to build a minimal image with:
+
+```$ make minimal```
+
+The `minimal` image doesn't have a frontend to update the user network proxy
+(`rush`) settings. Instead, one needs to use the `QOS_SPEC` environment variable
+to point to a configuration file. One way to add that configuration file is by
+creating a new image based on the `minimal` image.
+
```

Comment:
can we put an example config file here?  something like:
```
{
  "default_link": {
    "ingress": {
      "rate": ${INGRESS_BANDWIDTH},
      "loss": ${INGRESS_PACKET_LOSS},
      "latency": ${INGRESS_LATENCY},
      "jitter": ${INGRESS_JITTER},
      "jitter_strength": 0,
      "reorder_packets": false
    },
    "egress": {
      "rate": ${EGRESS_BANDWIDTH},
      "loss": ${EGRESS_PACKET_LOSS},
      "latency": ${EGRESS_LATENCY},
      "jitter": ${EGRESS_JITTER},
      "jitter_strength": 0,
      "reorder_packets": false
    }
  },
  "flows": []
}
```

Reply:
Great idea!